src/frontend/components/loginBanner.scss

```diff
@@ -0,0 +1,15 @@
+body{
```

Comment:
A body related styling should be moved to a root scoped css file not a component related one.